scripts/comfyui.py

```diff
@@ -60,14 +62,18 @@ def postprocess_batch_list(self, p, pp, *args, **kwargs):
         batch_results = external_code.run_workflow(
             workflow_type=default_workflow_types.postprocess_workflow_type,
             tab=self.get_tab(),
-            batch_input=list(pp.images),
+            batch_input=type_conversion.webui_image_to_comfyui(torch.stack(pp.images).to('cpu')),

```

Comment:
`torch.stack` here is wrong, it could raise an error if a previous extension already resized the batch

Reply:
Since comfyui works with batches, we could recover images that have the same shape and form groups of at most p.batch_size. Then for each of these groups, make a batch using `torch.stack` and call the comfyui workflow.